src/test/regress/expected/isolation_insert_vs_vacuum.out

```diff
@@ -0,0 +1,36 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1-begin s1-insert s2-vacuum-analyze s1-commit
+create_distributed_table
+
+               
+step s1-begin: 
+    BEGIN;
+
+step s1-insert: 
+    INSERT INTO test_insert_vacuum VALUES(1, 1);
+
+step s2-vacuum-analyze: 
+    VACUUM ANALYZE test_insert_vacuum;
+
+step s1-commit: 
+    COMMIT;
+
+
+starting permutation: s1-begin s1-insert s2-vacuum-full s1-commit
+create_distributed_table
+
+               
+step s1-begin: 
+    BEGIN;
+
+step s1-insert: 
+    INSERT INTO test_insert_vacuum VALUES(1, 1);
+
+step s2-vacuum-full: 
+    VACUUM FULL test_insert_vacuum;
+ <waiting ...>
```

Comment:
We should probably add a comment to the `LockRelationOid` call that we may take a stronger lock in the `ProcessUtility` in case of vacuum full.